docs/references/general/fitacf.md

```diff
@@ -1,2 +1,175 @@
-!!! Warning 
-    This page is under development. 
+<!-- Copyright (C) 2020 SuperDARN Canada, University of Saskatchewan 
+author(s): Marina Schmidt 
+
+Disclaimer: License under GNU v3.0, the file is found in the root directory under LICENSE 
+
+-->
+# FITACF files 
+
+FITACF files are a fitted post-processed data product from RAWACF files (sometimes dat files).
+
+## Naming Conventions
+
+Currently the common naming convention for FITACF files is:
+
+> YYYY.MM.DD.HH.mm.ss.<3-letter abbreviation>.fitacf
+
+Some radars operate using more than one channel and separate them by adding a letter to the file name:
+
+> YYYY.MM.DD.HH.mm.ss.<3-letter abbreviation>.[a-d].fitacf
+
+For example, on 2019-02-01 the King Salmon radar (KSR) was operating simultaneously on two channels as seen from the file names `20190201.0401.00.ksr.a.fitacf` and `20190201.0401.00.ksr.b.fitacf`.
+Each file contains data from a different frequency. 
+
+!!! Note
+    Sometimes modes like `twofsound` will write data into a single file. In this case the two frequencies are marked as two separate channels.
```

Comment:
```suggestion
    Sometimes modes like `twofsound` will write data into a single file. In this case the two frequencies are marked as two separate channels, denoted using the `chnum` parameter. However, it is important to note that some of the original SuperDARN radars were capable of operating in a `STEREO` mode (transmitting and receiving on 2 frequencies simultaneously), which was the original intended usage of the `chnum` parameter.
```

Reply:
Good addition, @asreimer . I suggest a slight change to clarify that there are still some stereo radars in the network (including newer ones)

```suggestion
Sometimes modes like `twofsound` will write data into a single file. In this case the two frequencies are marked as two separate channels, denoted using the `chnum` parameter. However, it is important to note that some SuperDARN radars have stereo capability (transmitting and receiving on 2 frequencies simultaneously), which was the original intended usage of the `chnum` parameter.
```

Reply:
Also, is it called "chnum" or "channel"? I can't see any field in the fitacf files called chnum.

Reply:
Oh, right, it's "channel" in fitted files and "chnum" in iqdat/rawacf. Oof, nice catch.

Reply:
I am committing @ecbland if that is okay. I have to pick one :p